tests/e2e/performance/test_pvc_bulk_creation_deletion_performance.py

```diff
@@ -111,14 +160,14 @@ def test_multiple_pvc_creation_after_deletion_performance(self, teardown_factory
         initial_number_of_pvcs = 120
         number_of_pvcs = math.ceil(initial_number_of_pvcs * 0.75)
 
-        log.info("Start creating new 120 PVCs")
+        log.info(f"Start creating new {initial_number_of_pvcs} PVCs in a bulk")
         pvc_objs = helpers.create_multiple_pvcs(
             sc_name=self.sc_obj.name,
             namespace=defaults.ROOK_CLUSTER_NAMESPACE,
             number_of_pvc=initial_number_of_pvcs,
             size=self.pvc_size,
             burst=True,
-        )
+        )[0]
```

Comment:
the function return a tuple, so you need to change to something like :
(pvc_objs,  placholder) = helpers.create_multiple_pvcs(...)

Reply:
Done, changed to pvc_objs, _ = helpers.create_multiple_pvcs(...)
Thanks